server/src/test/java/io/crate/types/TimeTZTypeTest.java

```diff
@@ -41,69 +46,69 @@ private static org.hamcrest.Matcher<TimeTZ> isTZ(long value, int zoneOffset) {
 
     @Test
     public void test_value_cast_null() {
-        assertThat(TimeTZType.INSTANCE.implicitCast(null), is(nullValue()));
+        assertThat(TimeTZType.INSTANCE.implicitCast(null, SESSION_SETTINGS), is(nullValue()));
     }
 
     @Test
     public void test_value_cast_not_null() {
         assertThat(
-            TimeTZType.INSTANCE.implicitCast(new TimeTZ(3600000000L, 7200)),
+            TimeTZType.INSTANCE.implicitCast(new TimeTZ(3600000000L, 7200), SESSION_SETTINGS),
             isTZ(3600000000L, 7200));
     }
 
     @Test
     public void test_implicit_cast_null() {
-        assertNull(TimeTZType.INSTANCE.implicitCast(null));
+        assertNull(TimeTZType.INSTANCE.implicitCast(null, SESSION_SETTINGS));
     }
 
     @Test
     public void test_implicit_cast_ISO_formats_with_time_zone() {
         assertThat(
-            TimeTZType.INSTANCE.implicitCast("01:00:00     CET"),
+            TimeTZType.INSTANCE.implicitCast("01:00:00     CET", SESSION_SETTINGS),
             Matchers.anyOf(isTZ(3600000000L, 3600), isTZ(3600000000L, 7200))
         );
-        assertThat(TimeTZType.INSTANCE.implicitCast("01:00:00     UTC"), isTZ(3600000000L));
-        assertThat(TimeTZType.INSTANCE.implicitCast("01:00:00     GMT"), isTZ(3600000000L));
-        assertThat(TimeTZType.INSTANCE.implicitCast("01:00:00  Z"), isTZ(3600000000L));
-        assertThat(TimeTZType.INSTANCE.implicitCast("01:00:00 +00"), isTZ(3600000000L));
-        assertThat(TimeTZType.INSTANCE.implicitCast("04:00:00-03:00"), isTZ(14400000000L, -10800));
-        assertThat(TimeTZType.INSTANCE.implicitCast("04:00:00+0300"), isTZ(14400000000L, 10800));
-        assertThat(TimeTZType.INSTANCE.implicitCast("04:00:00+03:00"), isTZ(14400000000L, 10800));
-        assertThat(TimeTZType.INSTANCE.implicitCast("04:00:00.123456789+03:00"), isTZ(14400123456L, 10800));
-        assertThat(TimeTZType.INSTANCE.implicitCast("04:00:00+0000"), isTZ(14400000000L));
-        assertThat(TimeTZType.INSTANCE.implicitCast("04:00:00.123456789-0000"), isTZ(14400123456L));
+        assertThat(TimeTZType.INSTANCE.implicitCast("01:00:00     UTC", SESSION_SETTINGS), isTZ(3600000000L));
```

Comment:
What do you think about defining an `implicitCast` method on each of those TypeTests, to avoid repeating the argument `SESSION_SETTINGS`?